Vostok.Metrics.AspNetCore/Helpers/EventHelper.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Diagnostics.Tracing;
+
+namespace Vostok.Metrics.AspNetCore.Helpers;
+
+internal static class EventHelper
+{
+    public static bool TryGetCounterValue(EventWrittenEventArgs eventData, string counterName, out long value)
+    {
+        value = 0;
+        if (eventData.Payload?.Count <= 0
+            || !(eventData.Payload?[0] is IDictionary<string, object> data)
+            || !data.TryGetValue("Name", out var n)
+            || !(n is string name)
+            || name != counterName) return false;
+
+        if (!data.TryGetValue("Mean", out var mean))
```

Comment:
That's quite confusing that we always check `mean` value.
Let's add a note about it here.